torch/csrc/distributed/rpc/message.h

```diff
@@ -8,6 +8,9 @@ namespace torch {
 namespace distributed {
 namespace rpc {
 
+// An enum denoting common RPC errors to allow specific error handling for them.
+enum RPCErrorType { UNKNOWN_ERROR = 0, TIMEOUT = 1, INTENTIONAL_FAILURE = 2 };
```

Comment:
what is meant by an "intentional failure"?
(even a 1-line comment might help clarify... is this like a user error, or an injected failure?)

Reply:
This is from the faulty PG agent, which can intentionally fail messages, so it is indeed an injected failure. Will add a comment to clarify